tests/data/sample.json

```diff
@@ -0,0 +1,63 @@
+{
+  "config": "tests/graphql_mock/test-custom-headers.graphql",
+  "name": "Example API",
```

Comment:
add proper names to files depicting what they are doing.

Reply:
done